core/unit_test/standalone/UnitTestMainInit.cpp

```diff
@@ -61,12 +61,15 @@
 #ifdef KOKKOS_ENABLE_HPX
 #include <hpx/TestHPX_Category.hpp>
 #endif
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+#include <openmptarget/TestOpenMPTarget_Category.hpp>
+#endif
 #ifndef TEST_EXECSPACE
 #ifdef KOKKOS_ENABLE_SERIAL
 #include <serial/TestSerial_Category.hpp>
 #endif
 #endif
-#include <TestStackTrace.hpp>
+#include <TestReducers_d.hpp>
```

Comment:
Did you mean to commit that change?

Reply:
No but it doesn’t matter. That file changes often anyway. Its not part of any automatic testing harness.